en/includes/guides/organization-management/share-applications.md

```diff
@@ -51,6 +67,59 @@ Follow the steps below to share a registered application with organizations.
 
     - While in the organization, select the shared application and ensure the username and password authenticator is enabled in the login flow. The organization users can then use the **Sign In with SSO** option to log in using their credentials.
 
+### Share application with all organizations
+
+Enable sharing for **all existing and future organizations**. New organizations automatically inherit shared access.
+
+![Share with all organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/share-with-all-organizations.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+**Why use this option?** When application's access is needed for all organizations without any restrictions.
+
+#### Advanced mode
+
+Share **specific roles only** with all organizations:
+
+1. Enable `Share a subset of roles with all organizations`
+
+2. Select the roles you want to share
+
+![Share a subset of roles with all organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/share-a-subset-of-roles-with-all-organizations.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+**Why use advanced mode?** This gives you precise control over application access. You select which roles to share initially, and these become the default for all organizations. Later, you can customize access by adding or removing specific roles for individual organizations.
+
+### Share application with selected organizations
+
+Enable sharing for **specific organizations only**. This gives you granular control over application access.
+
+![Share with selected organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/share-with-selected-organizations.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+**Why use this option?** Control exactly which organizations access your application and which roles they receive.
+
+Select **specific organizations** and choose which roles to share with each. Keep these rules in mind:
+
+- Share with **parent organizations first** before sharing with child organizations
+- Child organizations can only access roles available in their parent organization
+
+#### Sharing options
+
+You have two ways to configure role sharing for selected organizations:
+
+##### Option 1: Share all roles
+
+Click **Share All Roles** to automatically share every role with selected organizations.
+
+##### Option 2: Automatic inheritance
```

Comment:
Will it be okay to say this as inheritance? WDYT @AnuradhaSK

Reply:
fixed with 3e72de46ed6e33a31b5bfa74d5942d57d5ba529d